cloudferrylib/os/storage/cinder_storage.py

```diff
@@ -86,16 +86,17 @@ def cinder_client(self):
         return self.proxy(self.get_client(self.config), self.config)
 
     def get_client(self, params=None):
-        params = self.config if not params else params
+        params = params or self.config
 
-        return cinder_client.Client(
-            params.cloud.user,
-            params.cloud.password,
-            params.cloud.tenant,
-            params.cloud.auth_url,
-            cacert=self.config.cloud.cacert,
-            insecure=self.config.cloud.insecure
-        )
+        client_args = [params.cloud.user, params.cloud.password,
+                       params.cloud.tenant, params.cloud.auth_url]
+
+        client_kwargs = {"cacert": params.cloud.cacert,
+                         "insecure": params.cloud.insecure}
+        if params.cloud.region:
+            client_kwargs["region_name"] = params.cloud.region
+
+        return cinder_client.Client(*client_args, **client_kwargs)
```

Comment:
Why not pass None if no region is specified instead of this complex code with kwargs?